modules/psap-special-resource-operator-using-CRD.adoc

```diff
@@ -0,0 +1,451 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+:_content-type: PROCEDURE
+[id="deploy-simple-kmod-using-CRD"]
+= Building and running the simple-kmod SpecialResource for a hub and spoke topology
+
+You can use the Special Resource Operator (SRO) on a hub and spoke deployment in which Advanced Cluster Management connects a hub cluster to one or more managed clusters. This is possible in both connected and disconnected environments.
+This example shows how the Special Resource Operator builds driver containers in the hub, watching hub cluster resources to identify OCP versions for the helm charts that it uses to create resources which it delivers to spokes.
+
+[NOTE]
+====
+This procedure  can be used in both connected and disconnected environments. In the steps that follow, where indicated, you should choose the option appropriate to the environment you are working with.
+====
+
+Prerequisites
+
+* You have a running {product-title} cluster.
+* You installed the OpenShift CLI (`oc`).
+* You are logged into the OpenShift CLI as a user with `cluster-admin` privileges.
+* You installed the SRO.
+* You installed the Helm CLI (`helm`).
+* You installed the Advanced Cluster Management (ACM) operator.
+* You configured a container registry.
+
+.Procedure
+. Create a templates directory and change into it.
++
+[source,terminal]
+----
+$ mkdir -p charts/acm-simple-kmod-0.0.1/templates
+----
++
+[source,terminal]
+----
+$ cd charts/acm-simple-kmod-0.0.1/templates
+----
+. Create the buildconfig, policy, and placement.
+.. Save this YAML template for the image stream and build config in the `templates` directory as `0001-buildconfig.yaml`.
++
+[source,yaml]
+----
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+    labels:
+        app: {{ printf "%s-%s" .Values.specialResourceModule.metadata.name .Values.kernelFullVersion | replace "." "-" | replace "_" "-" | trunc 63 }}
+    name: {{ printf "%s-%s" .Values.specialResourceModule.metadata.name .Values.kernelFullVersion | replace "." "-" | replace "_" "-" | trunc 63 }}
+    annotations:
+specialresource.openshift.io/wait: "true"
+spec:
+    nodeSelector:
+        node-role.kubernetes.io/worker: ""
+    runPolicy: "Serial"
+    triggers:
+        - type: "ConfigChange"
+        - type: "ImageChange"
+    source:
+        git:
+            ref: {{.Values.git.ref}}
+            uri: {{.Values.git.uri}}
+        type: Git
+    strategy:
+        dockerStrategy:
+            dockerfilePath: Dockerfile.SRO
+            buildArgs:
+                - name: "IMAGE"
+                value: {{ .Values.driverToolkitImage  }}
+                {{- range $arg := .Values.buildArgs }}
+                - name: {{ $arg.name }}
+                value: {{ $arg.value }}
+                {{- end }}
+                - name: KVER
+                value: {{ .Values.kernelFullVersion }}
```

Comment:
indentation is off here, should be at the same level as "name".

Reply:
corrrected